changelog.d/20241112_110807_phil-PL-133007_nginx-dheat-mitigation_scriv.md

```diff
@@ -0,0 +1,21 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+- possibly impacts client connectivity to Nginx. Semi-official connectivity testers showed no change for popular clients and/or libraries but cannot cover every single implementation out there
+
+
+### NixOS XX.XX platform
+
+- restrict the Diffie-Hellman elliptic curves offered by Nginx to mitigate CVE-2024-41996
```

Comment:
Please include which exact curve is disabled, such that customers can determine whether they're affected or not from changelog information alone.